docker/fuzzbench/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
```

Comment:
Where is this going to run on cloud experiments?
I don't really like the idea of having the "brain" of the experiment running on a user's machine since these can go offline, run out of battery, have poor specs etc.

Reply:
It will run on the cloud, of course.

Reply:
great!